content/index.mdx

```diff
@@ -4,27 +4,53 @@ title: Interface guidelines
 
 import {HeroLayout} from '@primer/gatsby-theme-doctocat'
 export default HeroLayout
-
-Primer is a set of guidelines, principles, and patterns for designing and building UI at GitHub. It provides a shared language and standardized approach to delivering cohesive experiences.
-
-#### [Guides](/guides)
-
-Standards, guidelines, and tools to getting started with Primer.
-
-#### [Foundations](/foundations)
-
-The fundamental parts of the design system that underpin all GitHub interfaces, such as color and typography.
-
-#### [UI patterns](/ui-patterns)
-
-Design guidelines covering common user workflows.
-
-#### [Components](/components)
-
-Design and usage guidelines for individual Primer components.
-
-## Need help?
-
-If you found a bug on this website, please [open a new issue](https://github.com/primer/design/issues/new) with as much detail as possible, including steps to replicate the bug, screenshots, links, and expected results.
```

Comment:
Not sure where this should be 🤔

Reply:
Seems related to the open source section 🤔 

<img width="1026" alt="Screenshot 2023-07-21 at 09 55 44" src="https://github.com/primer/design/assets/912236/5c43324d-3cf0-4222-ad39-67ace0cc063f">